### ISSUE TITLE
Embed secure flow

### ISSUE THREAD

**Issue by eliawk** [Paratii-Video/paratii-portal#527]
@jellegerbrandy if a user has a secure wallet, how we keep it open in the embed player?
(atm the persistance is related to the `session` so in the embed)

**Comment by jellegerbrandy** [comment]
I don't get it. Can you rephrase the question? Which persistence are you talking about?

**Comment by jellegerbrandy** [comment]
I cannot confirm this behavior: visiting
https://portal.paratii.video/embed/JSeRyKaS0ebp does not require me to log
in

**Comment by eliawk** [comment]
@jellegerbrandy yes because atm the embed is not connected to secure flow, so it is always using the anonymous account.

**Comment by jellegerbrandy** [comment]
so I don't get your point? ARe you reporting a bug?

**Comment by jellegerbrandy** [comment]
closing this in favor of #630, that addresses this issue,
